src/data/finset/basic.lean

```diff
@@ -1947,6 +1947,8 @@ multiset.erase_dup_eq_self.2 (nodup_map_on H s.2)
 theorem image_id [decidable_eq α] : s.image id = s :=
 ext $ λ _, by simp only [mem_image, exists_prop, id, exists_eq_right]
 
+@[simp] theorem image_id' [decidable_eq α] : s.image (λ x, x) = s := image_id
```

Comment:
I'm guessing we can't just add a lemma that says `(λ x, x) = id` as that unifies too aggressively?

Reply:
I don't know.